crates/solver/src/solver/http_solver.rs

```diff
@@ -102,7 +106,13 @@ impl HttpSolver {
         gas_price: f64,
         external_prices: ExternalPrices,
     ) -> Result<(BatchAuctionModel, SettlementContext)> {
-        let tokens = map_tokens_for_solver(&orders, &liquidity);
+        let bufferable_token_list: HashSet<H160> = self
```

Comment:
I think I'd still keep using the name `market_makable_token_list` for consistency, instead of calling it `bufferable_token_list`.

Reply:
Fixed.